src/services/rsoServices.js

```diff
@@ -0,0 +1,21 @@
+export function getRsos() {
+    return ['Nebraskans For Cheese', 'Scott Frost Appreciation Club'];
+}
+
+export function getRsoInfo(rsoName) {
+    if (rsoName === 'Nebraskans For Cheese') {
+        return new Rso('Nebraskans For Cheese', 'Treasurer', 428);
```

Comment:
So the role is associated with the RSO itself rather than being a separate item? I guess this is OK for a kinda fake system :P

Reply:
Yeah, since we only have the one user it's the easiest solution.